MANIFEST.in

```diff
@@ -1,10 +1,11 @@
-include LICENSE.txt README CHANGES.txt
+include LICENSE.txt README.rst CHANGES.txt
 include requirements*.txt
+include dev.ini
```

Comment:
`dev.ini` is an example file, it is not necessary.

Also, "Fix MANIFEST.in" (as the commit message) may imply that it was broken. FUD :) I'd rather see something like "Be more explicit in MANIFEST.in"